modules/main/src/main/env/service/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
@felixgao  this looks to replicate ./bin/wasabi.sh test

if so, can we bring the concerns/objecties together?